ui/ui-frontend/projects/vitamui-library/src/app/modules/components/vitamui-sidenav-header/vitamui-sidenav-header.component.scss

```
@@ -0,0 +1,71 @@
+.vitamui-sidepanel-header {
+  display: flex;
+  flex-wrap: wrap;
+  background-color: white;
+  width: 100%;
+
+  .toolbar {
+    display: flex;
+    min-width: 100%;
+    width: 100%;
+    justify-content: flex-end;
+
+    .cancel {
+      > i {
+        padding: 0 1rem;
+        color: var(--vitamui-grey-600) !important;
+      }
+    }
+  }
+
+  .main-wrapper {
+    display: flex;
+    padding: 0 30px;
+    width: 100%;
+    max-width: 100%;
+
+    .title-wrapper {
+      max-width: 80%;
+      width: 80%;
```

Pour gérer le mode "wide" de la sidebar.

```suggestion
      flex-grow: 1;
      overflow: hidden;
```